tests/sandbox/LegacyBidirectionalIterator.h

```diff
@@ -0,0 +1,82 @@
+// FIXME remove pragma after moving to cts
+
+#pragma once
+
+#include "../common/common.h"
+
+#include "LegacyForwardIterator.h"
+#include "TypeTraits.h"
+#include "common.h"
+
+#include <type_traits>
+#include <utility>
+
+template <typename It>
+
+class legacy_bidirectional_iterator_requirement : requirement_verifier {
+ public:
+  bool check(It valid_iterator, size_t vector_size,
+             const std::string type_name) {
+    INFO("Verify named requiremnt Legacy Input Iterator for: " + type_name);
+    STATIC_CHECK(!std::is_same_v<It, void>);
+
+    legacy_forward_iterator_requirement<It>{}.check(valid_iterator, vector_size,

```

Comment:
I think we should wrap it into `verify` to make effect to the result of the verification.